pages/1.11/release-notes/1.11.0-rc1/index.md

```diff
@@ -2,7 +2,7 @@
 layout: layout.pug
 navigationTitle:  Release Notes for 1.11.0 Release Candidate 1
 title: Release Notes for 1.11.0 RC 1
```

Comment:
Should this be changed to 1.11.1?

Reply:
@orsenthil : No.

Reply:
Yup, got confused. Didn't see the file name.